src/cli/README.md

```diff
@@ -210,6 +212,64 @@ Set the Thread Child Timeout value.
 Done
 ```
 
+### commissioner start \<psdk\> \<provisioningUrl\>
+
+Start the Commissioner role.
+
+* pskd: Pre-Shared Key for the Joiner.
+* provisioningUrl: Provisioning URL for the Joiner (optional).
+
+This command will cause the device to send LEAD_PET and LEAD_KA messages.
+
+```bash
+> commissioner start ot-psk
+Done
+```
+
+### commissioner stop
+
+Stop the Commissioner role.
+
+This command will cuase the device to send LEAD_KA messages.
+
+```bash
+> commissioner stop
+Done
+```
+
+### commissioner energy \<mask\> \<count\> \<period\> \<scanDuration\> \<destination\>
+
+Send a MGMT_ED_SCAN message.
+
+* mask: Channels to perform IEEE 802.15.4 ED Scans.
+* count: Number of IEEE 802.15.4 ED Scans per channel.
```

Comment:
Ok, it seems the CLI doesn't allow setting of the duration, just the period between scans.